roles/kafka_controller/tasks/get_meta_properties.yml

```diff
@@ -19,3 +24,4 @@
   register: format_meta
   vars:
     clusterid: "{{ (zoo_cluster['content'] | b64decode).partition('cluster.id=')[2].partition('\n')[0] if kraft_migration|bool else uuid_key.stdout }}"
+  when: formatted.rc == 1 # To trigger the command only when the directories are not formatted
```

Comment:
is this some improvement or was it giving errors when ran on already formatted directories

Reply:
please refer to the ticket linked, I've added slack conversation

Reply:
got it